### PR TITLE
consolidate on express-handlebars for web + email

### PR DESCRIPTION
**Pull request by groovecoder**
Web templates were using express-hbs while email templates used
express-handlebars. Change both to use express-handlebars, which also
lets us share helpers between email + web templates.

### CODE REVIEW

**Comment by groovecoder** [comment]
Net minus 65 lines of code, and removes 8 dependencies! 🎉

File: controllers/user.js [hunk 1]
```diff
@@ -58,7 +51,7 @@ async function verify(req, res) {
     "report",
     {
       email: verifiedEmailHash.email,
-      date: new Date().toLocaleString("en-US", {year: "numeric", month: "long", day: "numeric"}),
+      date: HBSHelpers.prettyDate(new Date()),
```
Comment (groovecoder) [inline controllers/user.js]: Addresses https://github.com/mozilla/blurts-server/pull/330#discussion_r215278529

File: hbs-helpers.js [hunk 2]
```diff
@@ -1,87 +1,89 @@
...
-      return dataClasses.join(", ");
-    } else {
-      return "";
-    }
-  },
+function breachDataClasses(dataClasses) {
+  if (dataClasses.constructor === Array) {
+    return dataClasses.join(", ");
+  } else {
+    return dataClasses;
```
Comment (groovecoder) [inline hbs-helpers.js]: Addresses https://github.com/mozilla/blurts-server/pull/330#discussion_r215279775

File: views/email/email_partials/email_single_breach.hbs [hunk 3]
```diff
@@ -4,9 +4,9 @@
...
         <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; margin: 0px; font-weight: 600; font-size: 16px;">{{ Title }} Breach</p>
-        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Breach Date:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{ BreachDate }}</span></p>
+        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Breach Date:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{prettyDate BreachDate }}</span></p>
         <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Compromised Accounts:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{ PwnCount }}</span></p>
-        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Compromised Data:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{ DataClasses }}</span></p>
+        <p style="padding: 0px 0px 5px 0px; margin: 0px; color: #333333; font-weight: 600;">Compromised Data:<span style="font-weight:300; padding-left: 5px; color: #5d5d5d">{{breachDataClasses DataClasses }}</span></p>
```
Comment (pdehaan) [inline views/email/email_partials/email_single_breach.hbs]: Does this need an `{{#if DataClasses}}` check, similar to ./views/partials/breach_listing.hbs?

https://github.com/mozilla/blurts-server/blob/bbc186aebbcbf91ab94ca0b0cfbac55e22a0d5a7/views/partials/breach_listing.hbs#L9-L11

Comment (groovecoder) [reply views/email/email_partials/email_single_breach.hbs]: Good catch.

**Pull request merged**